client/main.cpp

```diff
@@ -0,0 +1,51 @@
+#include <boost/program_options.hpp>
+#include <iostream>
+#include "tcp_client.hpp"
+
+int main(int argc, const char *argv[])
+{
+    uint16_t port_num = 3333;
+    std::string ip_addr = "127.0.0.1";
+    namespace po = boost::program_options;
```

Comment:
Is this common practice to use `po` in the context of Boost? To me, `po` is something that diminishes the readability of the code.

Reply:
Done.